README.md

```diff
@@ -35,10 +35,14 @@ That's it! The only two things you need to take care about is keeping the LND no
 
 First of all make sure [Golang](https://golang.org/) and [Dep](https://github.com/golang/dep) are both correctly installed. Golang version 1.10 or newer is recommended.
 
+If you want to return a picture, make sure you install ImageMagick MagickWand C API. For examnple (Ubuntu/Debian): sudo apt-get install libmagickwand-dev
```

Comment:
Typo in `examnple`

And could you add a new line before the command and wrap "sudo apt-get install libmagickwand-dev" in these inline comment markers like in line 20?